api/nodes/views.py

```diff
@@ -1907,7 +1904,10 @@ def get_queryset(self):
                     del root_target.referent.node.commented_files[root_target._id]
 
             else:
-                url = waterbutler_api_url_for(self.get_node()._id, root_target.referent.provider, root_target.referent.path, meta=True)
+                referent = root_target.referent
+                if referent.provider == 'dropbox':
```

Comment:
The `StoredFileNode.path` for Dropbox includes the root folder, but waterbutler is expecting just the relative path otherwise it will throw a 404 error.